### PR TITLE
Somewhat standardized shop messages

### PR DESCRIPTION
**Pull request by MM-coder**
Started standardization, as per #5

### CODE REVIEW

File: commands/shop.py [hunk 1]
```diff
@@ -48,46 +48,56 @@ async def shop(self, ctx):
                         add_balance(ctx.author, -15.00)
                         await ctx.send("Purchase Successful, Please contact a member of staff to request your tag!")
                     else:
-                        await ctx.send("Insufficient Funds")
+                        em = discord.Embed(title = "Purchase Failed", description = "Insufficient Funds", color = colour.reds)
```
Comment (MattIBM) [inline commands/shop.py:51]: We use author fields to set embed titles, looks nicer in Discord

Comment (MattIBM) [reply commands/shop.py:51]: An Example
```
embed = discord.Embed(description=coinflip_string, color=colour.primary)
embed.set_author(name="Coinflip")
```

Comment (MattIBM) [reply commands/shop.py:51]: Also we don't wrap the `=` in spaces

Comment (MM-coder) [reply commands/shop.py]: Working on the `=` now

Comment (MM-coder) [reply commands/shop.py]: Fixed in b3bf5e3

**Comment by MM-coder** [comment]
I'm going to consider this ready for review since I don't see any other un-standardized messages

**Comment by MattIBM** [comment]
**Testing**

- Perhaps in the insufficient funds error message we could display how much the user has and how much is required? This isn't necessarily a bug and can be a task for 360

**Pull request merged**